src/api/services/participantsService.ts

```diff
@@ -279,17 +284,19 @@ export const updateParticipant = async (participant: Participant, req: UserParti
   } = updateParticipantSchema.parse(req.body);
   const { user } = req;
   const traceId = getTraceId(req);
+  const updatedParticipantId = Number(req.params.participantId);
 
   const auditTrailInsertObject = constructAuditTrailObject(
     user!,
     AuditTrailEvents.ManageParticipant,
     {
       action: AuditAction.Update,
-      apiRoles,
+      apiRoles: getRoleNamesByIds(apiRoles),
       participantName,
-      participantTypes: participantTypeIds,
+      participantTypes: mapClientTypeIdsToAdminEnums(participantTypeIds),
       crmAgreementNumber,
-    }
+    },
+    updatedParticipantId
```

Comment:
I intentionally left off the `participantId` from this action, because it probably shouldn't be shown to participants. e.g. it would expose things like crmAgreementNumber. 

Happy to discuss if you disagree!